package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "libvirt",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
@Rush nan2 targeted 0.10.29 at one point, the bug ticket about it seemed to indicate that older versions had an issue with utf8 but seems it might have been fixed. Have you tried building against 0.10?